specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-12-01-preview/examples/PutRegistrationDefinition.json

```diff
@@ -0,0 +1,92 @@
+{
+  "parameters": {
+    "scope": "subscription/0afefe50-734e-4610-8a82-a144ahf49dea",
+    "registrationDefinitionId": "26c128c2-fefa-4340-9bb1-6e081c90ada2",
+    "api-version": "2019-12-01-preview",
+    "requestBody": {
+      "properties": {
+        "registrationDefinitionName": "DefinitionName",
+        "description": "Tes1t",
+        "managedByTenantId": "83abe5cd-bcc3-441a-bd86-e6a75360cecc",
+        "authorizations": [
+          {
+            "principalId": "f98d86a2-4cc4-4e9d-ad47-b3e80a1bcdfc",
+            "roleDefinitionId": "acdd72a7-3385-48ef-bd42-f606fba81ae7",
+            "principalIdDisplayName": "principalIdDisplayName",
+            "roleAssignmentType": "Permanent",
+            "justInTimeAccessPolicy": {
```

Comment:
"justInTimeAccessPolicy" cannot be used in conjunction with "roleAssignmentType": "Permanent". Only when set to "Eligible".

Reply:
Thanks for catching that. Fixed in the new forced commit.